README.md

```diff
@@ -9,6 +9,11 @@ To open these notebooks in IPython, download the files to a directory on your co
 
 This will open a new page in your browser with a list of the available notebooks.
 
+Should this error `[TerminalIPythonApp] WARNING | File not found: u'notebook'` pop up, please install Jupyter by following the instructions [here](http://jupyter.readthedocs.io/en/latest/install.html) and execute the following command to run the notebook:
```

Comment:
"here" links are very bad style. You should consider simply putting the link under instructions. E.g., 
`...please install Jupyter by following the [instructions](http://jupyter.readthedocs.io/en/latest/install.html)...`